frappe/parallel_test_runner.py

```diff
@@ -134,8 +136,25 @@ def start_coverage(self):
 			]
 
 			if self.app == 'frappe':
```

Comment:
can we do this for ERPNext too? At least the part about excluding `test_*` files? 🤔

Reply:
If it helps to accurately calculate coverage 👍🏼

Reply:
Move ahead with merging? No one has stated any objection to this change.